src/sgame/sg_bot_util.cpp

```diff
@@ -2115,23 +2079,31 @@ bool BotEnemyIsValid( gentity_t *self, gentity_t *enemy )
 		return false;
 	}
 
-	//ignore buildings if we cant attack them
-	if ( enemy->s.eType == entityType_t::ET_BUILDABLE && ( !g_bot_attackStruct.integer ||
-	                                         self->client->ps.stats[STAT_CLASS] == PCL_ALIEN_LEVEL0 ) )
+	// ignore buildings if we can't attack them
+	if ( enemy->s.eType == entityType_t::ET_BUILDABLE && !g_bot_attackStruct.integer )
 	{
 		return false;
 	}
 
-	if ( BotGetEntityTeam( enemy ) == self->client->pers.team )
+	// dretch limitations
+	if ( self->client->ps.stats[STAT_CLASS] == PCL_ALIEN_LEVEL0 && G_DretchCanDamageEntity( self, enemy ) )
 	{
 		return false;
 	}
 
+	// ignore neutrals
 	if ( BotGetEntityTeam( enemy ) == TEAM_NONE )
 	{
 		return false;
 	}
 
+	// ignore teamates
+	if ( BotGetEntityTeam( enemy ) == BotGetEntityTeam( self ) )
```

Comment:
There is a G_OnSameTeam() function which could be used, here.
Also, is it really useful to split this check and previous one?

Reply:
I think the `G_OnSameTeam(team_t, team_t)` isn't merged yet?